app/assets/javascripts/news.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
Temporarily not, i would refer to my company's project, when i have a conclusion, i will tell you